test/integration/clientset_test.go

```diff
@@ -542,6 +542,15 @@ func testClusterServiceClassClient(sType server.StorageType, client servicecatal
 		return fmt.Errorf("should have two ClusterServiceClasses, had %v ClusterServiceClasses", len(serviceClasses.Items))
 	}
 
+	serviceClasses, err = serviceClassClient.List(metav1.ListOptions{FieldSelector: "metadata.name=" + name})
```

Comment:
Looks like we are missing existing tests for `ServiceClasses` and `ServicePlan` which were also modified in this PR. 

Let's use those existing tests in this file and add tests for these resources as well so that we can test all of your changes.

Reply:
I can't find any existing tests for these non-namespaced resources so just to clarify: would you like me to add tests for them in this PR? Or have I missed them somehow?

Reply:
I'm suggesting that you copy the existing test `testClusterServiceClassClient` and tweak it to test the namespaced resource `ServiceClasss` so that you can also test your changes made to that as well. Same for plans (`TestClusterServicePlanClient`).

Reply:
Done.